docs/operations.md

```diff
@@ -0,0 +1,96 @@
+# Operations on STM
+
+STMTools supports various operations on STM.
+
+## Enrich an STM
+
+Contextual data can be added to an STM by enrichment. At present, stmtools supports enriching an STM by static polygons.
+
+For example, if soil type data (`soil_map.gpkg`) is available together with an STM, then we can add the soil type and corresponding type code to the STM, using `enrich_from_polygon`:
+
+```python
+path_polygon = Path('soil_map.gpkg')
+fields_to_query = ['soil_type', 'type_code']
+stmat_enriched = stmat.stm.enrich_from_polygon(path_polygon, fields_to_query)
+```
+
+Two attributes from `soil_map.gpkg` to the STM: `soil_type` and `type_id`, will be added as data variables to the STM.
+
+When `soil_map.gpkg` is small, loading it as an `GeoDataFrame` is faster:
+
+```python
+import geopandas as gpd
+polygon = gpd.read_file('soil_map.gpkg')
+fields_to_query = ['soil_type', 'type_code']
+stmat_enriched = stmat.stm.enrich_from_polygon(path_polygon, fields_to_query)
+```
+
+## Subset an STM
```

Comment:
Just out of curiosity, how are these different from the standard Xarray filtering? If there are any advantages of doing these tasks via STMTools, maybe worth mentioning it?

Reply:
Indeed it's actually the same. We make it an option in "subset" to suit an existing behavior in their current MATLAB implementation.

I added the description of the equivalent filtering in Xarray. What we can do in the future is not forcing the evaluation of the mask. Documented this in issue #50